<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Downloader</class>
 <widget class="QWidget" name="Downloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Downloading...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QProgressBar" name="pageProgressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pageLabel">
     <property name="text">
      <string>Downloading pages list...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="picProgressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="picLabel">
     <property name="text">
      <string>Downloading images list...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="log"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
